airstack-modules/graphql/airstack-dynamic-nft-schema.graphql.ts

```diff
@@ -0,0 +1,79 @@
+const schema = `#--Airstack Schemas--
+
+enum AirProtocolType {
+    DYNAMIC_NFT
+}
+
+enum AirProtocolActionType {
+    UPDATE
+}
+
+type AirBlock @entity {
+    id: ID! #chain-number
+    hash: String!
+    number: BigInt!
+    timestamp: BigInt!
+}
+
+type AirMeta @entity {
+    id: ID! # air_meta
+    network: String!
+    schemaVersion: String!
+    slug: String! # POAP ??
+    name: String! # POAP
+    version: String!
+}
+
+type AirToken @entity {
+    id: ID!
+    address: String!
+}
+
+
+type AirEntityCounter @entity {
+    id: ID! #AIR_DYNAMIC_NFT_UPDATE_LAST_UPDATED_INDEX
+    count: BigInt!
+    createdAt: AirBlock!
+    lastUpdatedAt: AirBlock!
+}
+
+interface AirTransaction {
+    id: ID! 
+    logOrCallIndex: BigInt!
+    transactionHash: String! 
+    block: AirBlock!
+    index: BigInt!
+    protocolType: AirProtocolType!  #SOCIAL
+    protocolActionType: AirProtocolActionType!  #REGISTRATION
+}
+
+type AirDynamicNftUpdateTransaction implements AirTransaction @entity {
+    id: ID! #<chainId>-<transactionHash>-<logOrCallIndex>
```

Comment:
adding comments like 
```
"<chainId>-<transactionHash>-<logOrCallIndex>"
id: ID! 
```
so it'll show up on graphql interface

Reply:
Sure I will remove it